harvard-lts/drs-import-management-service
# Remove dataverse connection check

**Remove dataverse connection check**
* * *

**GitHub Issue**: [LTSEPADD-20](https://jira.huit.harvard.edu/browse/LTSEPADD-20)

# What does this Pull Request do?
This PR removes the check for connecting to dataverse in the health check. This is because we have brought dataverse down on dev and qa and DIMS does not successfully start unless it can connect to dataverse. DIMS should start independent of dataverse running.

# How should this be tested?

Running on dev - confirm DIMS is staying up

1. Pull PR branch
2. Build container locally 
3. docker compose -f docker-compose-dev.yml up -d --force-recreate
4. exec into container
5. Run `python3 -m unittest discover`

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? Yes
- integration tests? 

# Interested parties
@ives1227
